Models/Order.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BangazonAPI.Models
+{
+    public class Order
```

Comment:
The PR description states...

> In this PR, I have created the Department model, located in the Model folder. This model has no dependencies.

yet you have also submitted the Order model in this PR. Either update the description of the PR or remove this model from it